Python/samples/README.md

```diff
@@ -82,7 +91,7 @@ result = Recognizers.recognize_number("I have twenty apples")
 # 			"value": "20"
 # 		},
 # 		"text": "twenty",
-# 		"typeName": "number"
+# 		"type_name": "number"

```

Comment:
If this is happening on the output, this is breaking behaviour. Schema and naming must be the same across platforms.

Reply:
I guessed this (what `README.md` was showing) is the output of the "[sample script](https://github.com/daviddwlee84/Recognizers-Text/blob/master/Python/samples/simple_console/sample.py#L36)".
As you can see, this is basically the naming of the `ModelResult`'s attribute that I didn't change.
That is, this won't break any behavior, and instead, the old `README.md` is a little bit misleading.

Reply:
But the output of the sample script should be the same as the canonical representation of each result object. It currently doesn't behave correctly?

Reply:
I didn't change the `ModelResult`'s attribute. [It was named `type_name`](https://github.com/microsoft/Recognizers-Text/blob/master/Python/libraries/recognizers-text/recognizers_text/model.py#L16) since I used this package. Thus not sure if this should be `typeName` or not. And this is the result of how I got from running the sample script.

Reply:
Sorry, I was not aware of the discrepancy before and didn't have time to look into the issue in detail.
Can you make the output uniform? The objects returned by .NET and Python following the same schema.
Thanks for bringing this up while the Python package is still in alpha!

Reply:
No problem! You mean following [the naming of .NET](https://github.com/microsoft/Recognizers-Text/blob/master/.NET/Microsoft.Recognizers.Text/Models/ExtendedModelResult.cs) right?
But usually, in Python, people use PEP 8 more often that it said ["Use the function naming rules: lowercase with words separated by underscores as necessary to improve readability."](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables) (to make the code more "Python" :sweat_smile:)
So I did it in a compromised way that the `get_dict()` method returns `dict` with the same naming as .NET return. (If we change the schema of Python, that might influence lots of users currently)
Not sure which is the best way, looking for your opinion. :smile:

Reply:
We also use PEP8 in the project, so we'll need to add something to supress the naming issue just at this place.
Your ```get_dict()``` implements exactly what the behaviour should be. The key thing here is not so much the platform specifc naming conventions. Code in principle could follow whatever standard it needs.
But the returned object should be the same (for example, generate the same JSON, with the same schema, if serialized) and compatible between platforms. Which it currently is not, due to the issue you found.
I agree making this change will affect some users, but as the package is in alpha, this is the time to do it. We better be consistent from the start.

Reply:
I see. How about like change the [sample code](https://github.com/microsoft/Recognizers-Text/blob/master/Python/samples/simple_console/sample.py#L33-L38) into `print(result.get_dict())`. Or simply change `lambda o: o.__dict__` into `lambda o: o.get_dict()`
In this way, the attribute name of the Python object can still follow PEP 8, at the same time, the output can follow the platform convention.

Reply:
That's not enough, as serializing the result object will still have the problem. Same as deserializing the json specs into an Python object.

Reply:
Yeah, that makes sense. Do we want to modify this in this pull request or open another issue? This might have huge changes, that also influence unittests.

* https://github.com/microsoft/Recognizers-Text/blob/master/Python/tests/test_runner_number.py#L19
* https://github.com/microsoft/Recognizers-Text/blob/master/Python/libraries/recognizers-choice/recognizers_choice/choice/models.py#L33
* ...

Reply:
So that's how the tests currently pass. Not comparing the same names. :)
I agree. Let's make this change in a new PR. 
I'll do a final check on this one to merge. Thanks!